SumoLogic.Logging.Common/Sender/SumoLogicMessageSender.cs

```diff
@@ -182,7 +200,18 @@ public void TrySend(string body, string name)
 
             using (var httpContent = new StringContent(body, Encoding.UTF8, TextPlainMediaType))
             {
-                httpContent.Headers.Add("X-Sumo-Name", name);
+                if (!string.IsNullOrWhiteSpace(name))

```

Comment:
`if (!string` should be `if (!String` ?

Reply:
I googled the difference between `string` and `String` in C#. It says`string` is an alias for the `String` class in .NET framework. So technically there is no difference, but I'm wondering whether we have code style rules for this?

Reply:
why not be consistent with 2 usages below though?  Or change them all to `string`?

Reply:
Okay, I would change it to all `String`.

Reply:
the keyword `string` is an alias to the class `System.String` so it doesn't make a difference, besides style